dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -177,58 +182,50 @@ const buttonExpanded = css`
 	}
 `;
 
-const badgeDiameter = 20;
 const notificationColor = palette.error[400];
 
-const notificationBadgeStyles = css`
+const notificationBadgeStyles = (diameter: number) => css`
 	background-color: ${notificationColor};
 	color: ${palette.neutral[100]};
-	position: absolute;
-	top: 0;
-	left: 0;
-	min-width: ${badgeDiameter}px;
-	height: ${badgeDiameter}px;
-	border-radius: ${badgeDiameter / 2}px;
+	margin-top: 12px;
 	text-align: center;
 	display: flex;
 	justify-content: center;
 	align-items: center;
-	padding: 0 5px;
-	margin-left: -10px;
-	margin-top: -3px;
-`;
+	${textSans.xsmall()};
+	line-height: 1;
 
-const notificationStyles = css`
-	${textSans.xxsmall()};
-	color: ${notificationColor};
+	min-width: ${diameter}px;
+	height: ${diameter}px;
+	border-radius: ${diameter / 2}px;
 `;
 
-const redDotDiameter = 16;
-const redDotStyles = css`
-	background-color: ${notificationColor};
-	width: ${redDotDiameter}px;
-	height: ${redDotDiameter}px;
-	border-radius: ${redDotDiameter / 2}px;
+const dropdownButtonNotificationBadgeStyles = css`
 	position: absolute;
-	left: 5px;
-	top: 12px;
+	top: 0;
+	left: 0;
+	margin-left: -10px;
+	margin-top: -3px;
 `;
 
-const notificationCountStyles = css`
-	${textSans.xsmall()};
-	line-height: ${badgeDiameter}px;
+const notificationTextStyles = css`
+	${textSans.xxsmall()};
 `;
 
-const NotificationBadge = ({ count }: { count: number }) => {
+const NotificationBadge = ({
+	cssOverrides,
```

Comment:
Is there a way you could avoid overrides? They make testing and refactoring harder and you only have two use cases.

Reply:
👍